seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/TabPagerAdapter.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.seonoh.seonohapp
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+
+class TabPagerAdapter(
+    fm: FragmentManager
+) : FragmentStatePagerAdapter(fm) {
+
+    private lateinit var mData: ArrayList<String>
+
+    fun setData( data: ArrayList<String>){
+        mData = data
```

Comment:
addAll() 로 데이터를 받는게 좋을 것 같습니다
```suggestion
        mData.addAll(data)
        notifyDataSetChanged()
```

Reply:
네 수정했습니다!

Reply:
@seonoh 
반영 커밋 남겨주세요

Reply:
a2c58bd